client/my-sites/posts/post-list.jsx

```diff
@@ -277,7 +277,7 @@ var Posts = React.createClass( {
 				/>
 			);
 		} else {
-			if ( this.props.loading || ! this.props.sites.fetched ) {
+			if ( this.props.loading || ! this.props.haveSites ) {
```

Comment:
These are different things. User data may be stale and yet have posts, and vice-versa. Not sure what the best approach is here.